lib/smb2/packet/session_setup_request.rb

```diff
@@ -36,13 +36,10 @@ class SessionSetupRequest < Smb2::Packet
     }.freeze
     FLAG_NAMES = FLAGS.keys
 
-    def initialize(*args)
-      super
-      new_header = self.header
-      new_header.command = Smb2::COMMANDS[:SESSION_SETUP]
-      self.header = new_header
+    # @return [Symbol] a key in {Smb2::COMMANDS}
+    def self.command
```

Comment:
Change to `COMMAND = SESSION_SETUP` if you move to [class-constant lookup](https://github.com/rapid7/smb2/pull/28/files#r28820364)